FF Impressions (#6268)
braze-inc/braze-docs: _docs/_developer_guide/platform_wide/feature_flags/create.md

Do not set your rollout traffic above 0% until you are ready for your new feature to go live. When you initially define your feature flag in the dashboard, leave this setting at 0%.
{% endalert %}

## Check if the feature flag is enabled within your application {#enabled}

Once you have defined your feature flag, configure your app or site to check whether or not it is enabled for a particular user. When it is enabled, you'll set some action or reference the feature flag's variable properties based on your use case. The Braze SDK provides getter methods to pull your feature flag's status and its properties into your app. 

Feature flags are refreshed automatically at session start so that you can display the most up-to-date version of your feature upon launch. The SDK caches these values so they can be used while offline. 

{% alert note %}

> Be sure to log feature flag impressions 

Reviewer: Missing closing tag
```suggestion
Be sure to log [feature flag impressions](#impressions). 
{% endalert %}
```

{% endtab %}
{% endtabs %}

### Logging a feature flag impression {#impressions}

Track a feature flag impression whenever a user has had an opportunity to interact with your new feature, or when they __could__ have interacted if the feature is disabled (in the case of a control group in an A/B test).

> Usually, you can put this line of code directly underneath where you reference reference your feature flag:

Reviewer: ```suggestion
Usually, you can put this line of code directly underneath where you reference your feature flag in your app:
```

## Best practices

> ### Use either rollout or canvas

Reviewer: ```suggestion
### Don't combine rollouts with Canvases or experiments
```

> To avoid users being enabled and disabled by different entrypoints, you should either set the rollouts slider to a value greater than zero, OR enable the feature flag in a Canvas or experiment. As a best practice, if you plan to use a feature flag in a Canvas or experiment, keep the rollout percentage at zero.

Reviewer: ```suggestion
To avoid users being enabled and disabled by different entry points, you should either set the rollouts slider to a value greater than zero OR enable the feature flag in a Canvas or experiment. As a best practice, if you plan to use a feature flag in a Canvas or experiment, keep the rollout percentage at zero.
```
